src/app/util/attribute-storage.cpp

```diff
@@ -866,22 +866,6 @@ bool emberAfEndpointEnableDisable(EndpointId endpoint, bool enable)
         }
         else
         {
-            uint8_t i;
-            for (i = 0; i < emAfEndpoints[index].endpointType->clusterCount; i++)
-            {
-                const EmberAfCluster * cluster = &((emAfEndpoints[index].endpointType->cluster)[i]);
-                //        emberAfCorePrintln("Disabling cluster tick for ep:%d, cluster:0x%2X, %p",
-                //                           endpoint,
-                //                           cluster->clusterId,
-                //                           ((cluster->mask & CLUSTER_MASK_CLIENT)
-                //                            ? "client"
-                //                            : "server"));
-                //        emberAfCoreFlush();
-                emberAfDeactivateClusterTick(
```

Comment:
Don't we need some equivalent to cancel the timers for the relevant clusters for the relevant endpoint?

What is doing that now?  This needs to happen.

Reply:
created mechanism. Pls check https://github.com/project-chip/zap/pull/921